client/components/ConfigureTechnologyRow/index.jsx

```diff
@@ -78,116 +81,124 @@ class ConfigureTechnologyRow extends Component {
         deleteTechnologyRow(index);
     }
 
+    undoDelete() {
+        const { undoDeleteTechnologyRow, index } = this.props;
+        undoDeleteTechnologyRow(index);
+    }
+
     render() {
         const {
             availableBrowsers,
             availableAts,
             runTechnologies,
             index,
-            editable
+            editable,
+            deleted
         } = this.props;
 
-        return (
-            !editable ? (
-                <tr>
-                    <td>
-                        {
-                            availableAts.find(
-                                at => at.at_id === runTechnologies.at_id
-                            ).at_name
+        return !editable ? (
+            <tr>
+                <td>
+                    {
+                        availableAts.find(
+                            at => at.at_id === runTechnologies.at_id
+                        ).at_name
+                    }
+                </td>
+                <td>{runTechnologies.at_version}</td>
+                <td>
+                    {
+                        availableBrowsers.find(
+                            browser => browser.id === runTechnologies.browser_id
+                        ).name
+                    }
+                </td>
+                <td>{runTechnologies.browser_version}</td>
+                <td>
+                    <Button
+                        variant="danger"
+                        aria-label={`Delete at/browser combination ${index +
+                            1}`}
+                        onClick={this.deleteRun}
+                        disabled={deleted === true}
+                    >
+                        Remove
+                    </Button>
+                    <Button
+                        disabled={deleted === false}
+                        onClick={this.undoDelete}
```

Comment:
we need an "aria-label" here that says "Undo delete", otherwise there is no way for a screen reader user ot know what this button does!